tests/unit/actions/indexables-page-action-test.php

```diff
@@ -3,6 +3,7 @@
 namespace Yoast\WP\SEO\Tests\Unit\Actions;
 
 use Mockery;
+use Prophecy\PhpDocumentor\ClassAndInterfaceTagRetriever;
```

Comment:
What's this?

Reply:
My best guess is that I misspelled something in VSCode, it corrected it to this `ClassAndInterfaceTagRetriever` and automatically added the missing dependency. How cool is this? 😁

Reply:
computers are so clever and so dumb